manifests/agent/config.pp

```diff
@@ -2,7 +2,9 @@
 
   Ini_setting {
     path    => $puppet::conf,
+    section => 'agent',
     ensure  => 'present',
+    notify  => Service['puppet_agent'],
```

Comment:
but as I said, if we use cron then this notify schedules a refresh but it doesn't actually start the service, thus it's not a problem